### ISSUE TITLE
iptable mangle rules for webui port (8080) seem to block communication to trackers listening on 8080

### ISSUE THREAD

**Issue by sebguilbaud** [DyonR/docker-qbittorrentvpn#60]
no trackers listening to 8080 port work until I remove the "dst 8080" rule in the OUPUT chain of the mangle table

**Comment by D1s0rdu8** [comment]
Same issue here, can you elaborate the method to manually remove the entry in the mangle table ?

**Comment by sebguilbaud** [comment]
It's no longer needed with recent versions

**Comment by D1s0rdu8** [comment]
Yes, I don't have mange entries, but still, the issue persist with 8080 trackers

**Comment by sebguilbaud** [comment]
` docker exec -it <name of container> iptables -xnvL -t mangle` should show you rules eventually mangling 8080/tcp port

**Comment by D1s0rdu8** [comment]
Thank you.
I can confirm that there is no 8080 mangle rules applied when using dyonr/qbittorrentvpn:dev 
For whatever other reasons trackers using 8080 are still not working.
